app/src/main/java/com/mk/tasky/agenda/presentation/home/AgendaItemMapper.kt

```diff
@@ -0,0 +1,32 @@
+package com.mk.tasky.agenda.presentation.home
+
+import com.mk.tasky.agenda.domain.model.Agenda
+import com.mk.tasky.agenda.domain.model.Reminder
+import com.mk.tasky.agenda.domain.model.Task
+
+// TODO: Add Event
+fun Agenda.toAgendaItem(): List<AgendaItem> {
+    return this.reminders.map { it.toAgendaItem() } + this.tasks.map { it.toAgendaItem() }
```

Comment:
Is this only related to presentation? Might rather put this in domain or data directly and expose a list of `AgendaItems` to presentation

Reply:
Removed it after the migration to AgendaItem, since it didn't made much sense